boxnav/boxsim.py

```diff
@@ -56,20 +57,36 @@ def simulate(args: Namespace) -> None:
         action_taken, _ = agent.execute_navigator_action()
         print("Action taken:", action_taken)
 
-        actions_pbar.update()
-
         # Navigation progress is based on the percentage of the environment navigated
-        navigation_pbar.count = int(agent.get_percent_through_env())
-        navigation_pbar.update()
+        actions_pbar.update()
 
     if args.ue:
         agent.ue.close_osc()
 
-    navigation_pbar.close()
     actions_pbar.close()
 
     print("Simulation complete.")
 
+    if args.auto_upload:
+        chdir("..")
+        chdir("learning")
+        sprun(
+            [
+                "python",
+                "upload_data.py",
+                PurePath(agent.image_directory).stem,
+                args.auto_upload,
+                (
+                    "Automatically uploading data from "
+                    + PurePath(agent.image_directory).stem
+                    + " run."
```

Comment:
Use an f-string.

Reply:
I used f-strings in other places as needed as well.